Print exception stack trace in CsPackageManager.TryRun

Honkbinger opened the pull request:
This is not very helpful
![Screenshot_4](https://github.com/user-attachments/assets/b790d26f-143a-422d-8668-8e1f8f55e122)
I suggest replacing it with at least this
![Screenshot_3](https://github.com/user-attachments/assets/7f8f9dff-2d41-4e1c-98fe-e0e771d0ea57)
### Notes
You can just print e, but it's a bit ugly:
![Screenshot_5](https://github.com/user-attachments/assets/6dcf31f2-af67-41d6-8e88-533710c46ee5)
Idk how to capture line number
Also, in theory there's a simpler method:
```C#
StackTrace st = new StackTrace(e);
st = new StackTrace(st.GetFrames().SkipLast(1));
```
But it's not available in .NET 6 :BaroDev:

Honkbinger committed:
print(e.Message) => print(e)

Honkbinger committed:
Print exception stack trace in CsPackageManager.TryRun

Honkbinger committed:
added Inner Exception back

Honkbinger commented:
At first i thought inner exception could never happen and is useless but now i found that exceptions in static constructors could trigger it
I added some prints but i didn't test them yet
I obviously tested it by just replacing TryRun with harmony, but i can't do that for inner exception because it happens before my testing mod initialize :BaroDev:
So i guess i need to compile luatrauma to test it, and it'll probably take me a few hours to figure out how to compile it in VSCode
:BaroDev:

MapleWheels commented:
Hi Honk,

This is a good PR. I won't be able to merge it as we're doing a big code refactor. However, I will be manually implementing this into the Runner. Thanks for the contribution.

I'm going to leave the PR open for now and will close it once implemented.

Honkbinger committed:
skip 2 frames

Finally was able to compile luatrauma
And stack trace seems to be 1 frame longer than in harmony patch
also you need System.Diagnostics (kek)
